File: CForces/705A.py
n = int(input())
f = True
s = 'I hate '
for i in range(n - 1):
    if f: 
        s += 'that I love '
        f = False
    else:
        s += 'that I hate '
        f = True
s += 'it' 
print(s)

File: CForces/705A.cpp
#include <iostream>
#include <string>
using namespace std;

int main(){
    int n;  
    cin >> n;
    bool s = true;
    string end = "it";
    string phrase = "I hate "; 
    string hate = "that I hate ";
    string love = "that I love ";
    for(int i = 1; i < n; i++){
       if(s){
          phrase += love; 
          s = false;
       }
       else{
           phrase += hate;
           s = true;
       }
    }
    phrase += end;
    cout << phrase << endl;
}
